templates_cspm_eventbridge/OrgFullInstall.yaml

```diff
@@ -90,9 +87,8 @@ Resources:
                 - !GetAtt AdministrationRole.RoleId
             Action:
               - sts:AssumeRole
-      Path: /
       ManagedPolicyArns:
-        - !Sub arn:${AWS::Partition}:iam::aws:policy/AdministratorAccess
```

Comment:
this was taken from AWS docs https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/stacksets-prereqs-self-managed.html. but the change also looks good.